src/controllers/patient-controller.ts

```diff
@@ -18,7 +18,7 @@ export const createPatient = async (_req: Request, res: Response, _next: NextFun
     const newPatient = await PatientModel.create(_req.body);
```

Comment:
why is there a underscore here in _req

Reply:
I used it casually just to name the params, didnt know the underscore prefixed variable denote they are not used in function scope